src/main/resources/templates/index.html

```diff
@@ -194,7 +194,7 @@ <h3 class="govuk-heading-m" style="display:inline" th:text="${filing.companyName
                     </div>
                     <div class="govuk-grid-column-one-third" style="text-align: right;">
                         <div class="govuk-button-group" style="justify-content: flex-end;margin-bottom: 0;">
-                            <a th:href="${filing.getViewerLink()}" role="button" draggable="false" class="govuk-button" data-module="govuk-button" data-i18n="index:results.downloads.openViewer">
+                            <a th:href="${filing.getViewerLink()}" role="button" draggable="false" class="govuk-button" data-module="govuk-button" data-i18n="index:results.downloads.openViewer" target="_blank" rel="noopener noreferrer">
```

Comment:
Are `nooopener` and `noreferrer` achieving something specific and intentional here?

Reply:
From what I understand they are there to sever and logical connection between the referring page and the newly opened page.  I added it intentionally as it seems to be a best practice even though the UK iXBRL Viewer link basically contains the same information.

Reply:
I _think_ it's mostly for opening links to external sites, but I don't feel strongly about excluding it. I just worry that there's unintended side effects there in exchange for no obvious benefit.